File: 程式語言 C V2答案檔/307.c
#include <stdio.h>
#include <stdlib.h>

int compute(int a[]);
int main()
{
    int arr[5], i, maximum;
    
    for(i=0; i<5; i++)
        scanf("%d", &arr[i]);

    maximum = compute(arr);
    printf("%d\n", maximum);
    return 0;
}

// ===================================================
// Please write your code in the specified function
// Do NOT change the function name, type of parameter
// ===================================================
int compute(int a[])
{
    int k, M;
    M=a[0];
    for(k=0; k<5; k++)
        if (a[k]>M)
            M=a[k];
    return M;
}

File: 物件導向程式語言 C# 7答案檔/307.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSA03
{
    class CSA03
    {

        static void Main(string[] args)
        {
            try
            {
                List<string> lst = new List<string>();
                string str = "";
                while (true)
                {
                    str = Console.ReadLine();
                        if (str != "999")
                        {
                            lst.Add(str);
                    }
                    else
                    {
                        break;
                    }
                }
                compute(lst);
            }
            catch
            {
                Console.Write("error");
            }
            Console.ReadKey();
        }
        // ==================================================
        // Please write your code in the specified Method
        // Do NOT change the method name, type of parameter
        // ==================================================
        static void compute(List<string> lst)
        {
            Console.WriteLine("before:" + String.Join(" ", lst));
            lst.Reverse();
            Console.WriteLine("after:" + String.Join(" ", lst));
        }
  
    }
}
